mkt/commonplace/management/commands/deploy_build_id.py

```diff
@@ -3,10 +3,10 @@
 import sys
 
 from django.conf import settings
-from django.core.files.storage import default_storage as storage
 from django.core.management.base import BaseCommand
 
 from mkt.commonplace.models import DeployBuildId
+from mkt.site.storage_utils import local_storage
```

Comment:
I love that we have these explicit storage classes now. private for what must be private, public for what must be public, local for what absolutely needs to stay local, it's awesome. :+1: